packages/docsearch-react/src/useDocSearchKeyboardEvents.ts

```diff
@@ -39,7 +39,8 @@ export function useDocSearchKeyboardEvents({
       if (
         (event.keyCode === 27 && isOpen) ||
         // The `Cmd+K` shortcut both opens and closes the modal.
-        (event.key.toLowerCase() === 'k' && (event.metaKey || event.ctrlKey)) ||
+        (event.key?.toLowerCase() === 'k' &&
```

Comment:
I suppose this would explain what can be going wrong?

```suggestion
        // We need to check for `event.key` because it can be `undefined` with
        // Chrome's autofill feature.
        (event.key?.toLowerCase() === 'k' &&
```

Reply:
I'm surprised that this happens in DocSearch though, because DocSearch shouldn't let the Chrome autofill behavior trigger. Do you have a reproduction where I can see it happen with DocSearch?

Reply:
Not DocSearch itself.
https://github.com/algolia/docsearch/blob/bae117ce6db0e51f7c0958215705dc849b25212c/packages/docsearch-react/src/useDocSearchKeyboardEvents.ts#LL67C51-L67C51
`keydown` is listening globally, so my component will cause this issue.

Reply:
Ah got it. You can accept my suggestion above and I'm good with that change 👍